engine/src/main/scala/cromwell/webservice/metadata/MetadataBuilderActor.scala

```diff
@@ -131,7 +133,9 @@ object MetadataBuilderActor {
 
   /** Sort events by timestamp, transform them into TimestampedJsValues, and merge them together. */
   private def eventsToIndexedJson(events: Seq[MetadataEvent]): TimestampedJsValue = {
-     events map { e => keyValueToIndexedJson(e.key.key, e.value, e.offsetDateTime) } reduce(_ |+| _)
+    // The `List` has a `Foldable` instance defined in scope, and because the `List`'s elements have a `Monoid` instance
+    // defined in scope, `suml` can derive a sane `TimestampedJsValue` value even if the `List` of events is empty.
+    events.toList map { e => keyValueToIndexedJson(e.key.key, e.value, e.offsetDateTime) } suml
```

Comment:
Yes you're right that `suml` is implemented as `foldLeft` using the `Monoid.zero` for the initial value.  I made this change because the current `reduce` code crashes when the list is empty (not just because moar category theory 😄 ).  We can certainly revisit this later if it turns out to warrant a parallel implementation.